src/Viewer.tsx

```diff
@@ -106,6 +112,7 @@ function Viewer(): ReactElement {
   const [currentFrame, setCurrentFrame] = useState<number>(0);
   /** Backdrop key is null if the dataset has no backdrops, or during initialization. */
   const [selectedBackdropKey, setSelectedBackdropKey] = useState<string | null>(null);
+  const annotationState = useAnnotations();
```

Comment:
Hook introduced in annotations pt. 1! It stores all the annotation-related state :)